tests/manage/pv_services/pvc_snapshot/test_rbd_block_pvc_snapshot.py

```diff
@@ -0,0 +1,204 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import skipif_ocs_version, ManageTest, tier1
+from tests.helpers import wait_for_resource_state, create_pods
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@skipif_ocs_version('<4.6')
+@pytest.mark.polarion_id('OCS-2361')
+class TestRbdBlockPvcSnapshot(ManageTest):
+    """
+    Tests to take snapshots of RBD Block VolumeMode PVCs
+
+    """
+    @pytest.fixture(autouse=True)
+    def setup(
+        self, project_factory, snapshot_restore_factory,
+        multi_pvc_factory, pod_factory
+    ):
+        """
+        Create PVCs and pods
+
+        """
+        self.pvc_size = 5
+
+        self.pvc_objs = multi_pvc_factory(
+            interface=constants.CEPHBLOCKPOOL,
+            size=self.pvc_size,
+            access_modes=[
+                f'{constants.ACCESS_MODE_RWX}-Block',
+                f'{constants.ACCESS_MODE_RWO}-Block'
+            ],
+            status=constants.STATUS_BOUND,
+            num_of_pvc=2,
+            wait_each=False
+        )
+
+        self.pod_objs = create_pods(
+            self.pvc_objs, pod_factory, constants.CEPHBLOCKPOOL,
+            pods_for_rwx=1, status=constants.STATUS_RUNNING
+        )
+
+    def md5sum(self, pod_obj):
```

Comment:
can we use the existing ocs_ci.ocs.resources.pod.cal_md5sum?

Reply:
Yes, the required code change is merged in another PR. I will update this.

Reply:
Done